dot/network/proto/api.v1.pb.go

```diff
@@ -1,15 +1,17 @@
+// Copyright 2021 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
 // Schema definition for block request/response messages.
 
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.23.0
-// 	protoc        v3.14.0
+// 	protoc-gen-go v1.28.1
+// 	protoc        v3.21.9
```

Comment:
Should we bump these in a separate PR instead?

Reply:
Created #2992 to fix the proto generation and bump versions, but we can merge your PR first, it's fine with me since it's all broken on development already